src/utils/update/managing.officer.mapper.ts

```diff
@@ -31,15 +31,15 @@ export const mapToManagingOfficer = (officer: CompanyOfficer): ManagingOfficerIn
 };
 
 export const mapToManagingOfficerCorporate = (officer: CompanyOfficer): ManagingOfficerCorporate => {
-  const service_address = mapBOMOAddress(officer.address);
+  const service_address = mapBOMOAddress(undefined);
```

Comment:
We'll have no addresses displayed for corporate managing officer if we set both service_address and principle_address to undefined?

Reply:
Yeah I also didn't understand we'd do this and asked @cdrainCH to clarify with Matthew Poland what the expectation is here ..

Reply:
So early on Friday Matthew thought that this was the right path to take, but after a chat with Stephanie he realised that this wasn't necessary so this will be reverted and the address will be mapped to the service address again

Reply:
Yeah, Matthew and I ran through this Friday evening. The only change required is not selecting the yes/no radio button. Let's please remember to do 3 amigos when picking up a ticket as it would have saved some time and confusion here :)